Dockerfile.rie

```diff
@@ -0,0 +1,19 @@
+FROM public.ecr.aws/lambda/ruby:3.3.2025.08.26.15
```

Comment:
let's use `public.ecr.aws/lambda/ruby:3.3` as this tag will be overiden every time we will release a new image for 3.3.